extensions/wikia/ContentReview/controllers/SpecialContentReviewController.class.php

```diff
@@ -0,0 +1,24 @@
+<?php
+use  Wikia\ContentReview\Models\ReviewModel;
+
+class SpecialContentReviewController extends WikiaSpecialPageController {
+
+	function __construct() {
+		parent::__construct( 'ContentReview', 'ContentReview', true );
+	}
+
+	public function index() {
+		$this->wg->Out->setPageTitle( wfMessage( 'content-review-special-title' )->escaped() );
+
+		$this->reviewStatus = [];
```

Comment:
There is no need to initialize this variable while you are assigning some value underneath.